students/Jaidjen/lesson03/SlicingLab.py

```diff
@@ -0,0 +1,27 @@
+
+def first_and_last(seq):
+      return seq[-1:] + seq[1:-1] + seq[:1]
+myseq = first_and_last("Reverse the order of words")
+print(myseq)
+
+def rem_item(seq):
+       return seq[::2]
+myseq1 = rem_item("Reverse the order of words")
+print(myseq1)
+
+def rem_four(seq):
+        return seq[4:-4:2]
+myseq2 = rem_four("Reverse the order of words")
+print(myseq2)
+
+def swap_words(seq):
+      return(' '.join(seq.split()[::-1]))
```

Comment:
the direction is to return the same sequence - don't use join or split as it will error on tuple and list

the assignment said that this should work for tuple, list and strings.